### PR TITLE
[T7][F12-B1]

### PR DESCRIPTION
**Pull request by yeejfe**
Updated user guide and developer guide
git tag as V0.0.

### CODE REVIEW

**Comment by yeejfe** [comment]
All of us contributed in V0.0.
We edited the markdown files in hackmd.io for easier and quicker integration,

**Comment by yeejfe** [comment]
Ready for review!

Just remember the above...

**Comment by yeejfe** [comment]
Revert changes, because of clicking the wrong branch for commit

File: README.md [hunk 1]
```diff
@@ -1,23 +1,18 @@
-# Address Book (Level 4)
+# Savvy To-Do
```
Comment (brandonyeoxg) [inline README.md:1]: Good job changing the title.

File: README.md [hunk 2]
```diff
@@ -1,23 +1,18 @@
...
 
-[![Build Status](https://travis-ci.org/se-edu/addressbook-level4.svg?branch=master)](https://travis-ci.org/se-edu/addressbook-level4)
-[![Build status](https://ci.appveyor.com/api/projects/status/3boko2x2vr5cc3w2?svg=true)](https://ci.appveyor.com/project/damithc/addressbook-level4)
-[![Coverage Status](https://coveralls.io/repos/github/se-edu/addressbook-level4/badge.svg?branch=master)](https://coveralls.io/github/se-edu/addressbook-level4?branch=master)
-[![Codacy Badge](https://api.codacy.com/project/badge/Grade/fc0b7775cf7f4fdeaf08776f3d8e364a)](https://www.codacy.com/app/damith/addressbook-level4?utm_source=github.com&amp;utm_medium=referral&amp;utm_content=se-edu/addressbook-level4&amp;utm_campaign=Badge_Grade)
+[![Build Status](https://travis-ci.org/CS2103JAN2017-F12-B1/Main.svg?branch=master)](https://travis-ci.org/CS2103JAN2017-F12-B1/Main)
```
Comment (brandonyeoxg) [inline README.md:3]: Glad to see travis was set up correctly =).

**Review by brandonyeoxg — COMMENTED** [review]
@yeejfe Accepting this PR without a review because another of your PRs for this tutorial was reviewed in detail

**Pull request closed without merging**